app/javascript/packs/document-capture-welcome.js

```diff
@@ -0,0 +1,10 @@
+import { isCameraCapableMobile } from '@18f/identity-device';
+
+const form = document.querySelector('form[action="/verify/doc_auth/welcome"]');
```

Comment:
WDYT of using an ID or a tag or something on the form, instead of the action URL? If we changed the URL I would not expect to check here

I typically add class names with `js-` so I know the correspond to something in JS, so if we did something like `form.js-skip-upload-check` or something?

Reply:
> WDYT of using an ID or a tag or something on the form, instead of the action URL? If we changed the URL I would not expect to check here
> 
> I typically add class names with `js-` so I know the correspond to something in JS, so if we did something like `form.js-skip-upload-check` or something?

Good idea 👍  I'll make the change.

Reply:
> > WDYT of using an ID or a tag or something on the form, instead of the action URL? If we changed the URL I would not expect to check here
> > I typically add class names with `js-` so I know the correspond to something in JS, so if we did something like `form.js-skip-upload-check` or something?
> 
> Good idea 👍 I'll make the change.

Updated in rebased 0b74ac5.